components/Archetype/basic_usage.rst

```diff
@@ -0,0 +1,28 @@
+Basic Usage
+===========
+
+.. code-block:: php
+
+    // File example: src/script.php
+    <?php
+
+    // update this to the path to the "vendor/" directory, relative to this file
+    require_once __DIR__ . '/vendor/autoload.php';
```

Comment:
We decide not use:
``  // File example: src/script.php ``
`` <?php``
``// update this to the path to the "vendor/" directory, relative to this file``
`` require_once __DIR__ . '/vendor/autoload.php'; ``
istead of this use `` <?``